src/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 0;
+}
+
+.card_img {
+  width: 160px;
+  margin-bottom: 40px;
+}
+
+.card_text {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card_product_code {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card_stars_reviews {
+  display: flex;
+  margin-bottom: 24px;
+  justify-content: space-between;
+}
+
+.card_reviews {
+  color: #060b35;
+  font-family: "Roboto", sans-serif;
```

Comment:
no need to add font-family for each element.
just set it in html selector one time